Sources/CustomDump/Dump.swift

```diff
@@ -134,11 +136,22 @@ public func customDump<T, TargetStream>(
 
     case let (value as AnyObject, .class?):
       let item = ObjectIdentifier(value)
+      var occurence = occurencePerType[typeName(mirror.subjectType), default: 0] {
+        didSet { occurencePerType[typeName(mirror.subjectType)] = occurence }
+      }
+
+      var id: String {
+        let id = idPerItem[item, default: occurence]
+        idPerItem[item] = id
+
+        return id > 1 ? "<\(id)>" : ""
```

Comment:
One thought (suggested by Brandon) is to maybe use `#` instead of `<>` to avoid clashing with generic parameter list syntax:

```suggestion
        return id > 1 ? "#\(id)" : ""
```

Thoughts? Any concerns with such a change?

Reply:
I think this makes sense

Reply:
Done